### PR TITLE
UPD: adding trim to go build

### PR DESCRIPTION
**Pull request by wildy862**

### CODE REVIEW

**Comment by aandrew-me** [comment]
Thanks for your merge requests.

File: build.sh [hunk 1]
```diff
@@ -2,14 +2,13 @@
 
 # For GNU Linux
 go build -ldflags="-s -w" -o ./build/tgpt-linux-amd64
```
Comment (aandrew-me) [inline build.sh:4]: Is there any particular reason for omitting this line?

Comment (wildy862) [reply build.sh]: sorry i forget add to this line :)

**Review by wildy862 — COMMENTED** [review]
Done

**Comment by aandrew-me** [comment]
Great! Wish you all the best!

**Pull request merged**